Backend/src/main/java/org/eclipse/cbi/StatefulSetData.java

```diff
@@ -0,0 +1,38 @@
+package org.eclipse.cbi;
+
+public class StatefulSetData {
+    private String name ;
+    private String uid ;
+    private int replicas ;
+    private int readyReplicas ;
+    
+    public String getName() {
+        return(name);
+    }
+    public String getUid() {
+        return(uid);    
+    }
+    public String getReplicas() {
+        return(name);
```

Comment:
should return "replicas"

Reply:
They are not being used right now so I am removing them